annex/chapter1/adv_examples/src/position_definition.cpp

```diff
@@ -0,0 +1,15 @@
+#include <numeric>
```

Comment:
rename to position.h and move to the include folder. We never include files from the src folder.

Reply:
Done